Makefile

```diff
@@ -110,6 +110,42 @@ fmt refmt::
 fmt-no-promote refmt-no-promote::
 	@esy dune build @fmt
 
+
+new-openbsd:
+	ulimit -s 10000
+	doas ftp -o /usr/local/bin/shasum https://fastapi.metacpan.org/source/MSHELOR/Digest-SHA-6.02/shasum
+	doas chmod +x /usr/local/bin/shasum
+	doas mkdir -p /app/esy-install && \
+		doas chown -R $(USER):$(USER) /app/esy-install
+	gmake -E "APP_ESY=/app/esy" -E "APP_ESYINSTAL=/app/esy-install/" new-docker 
+
+APP_ESY ?= /app/esy
```

Comment:
Why is this hardcoded to `/app/esy` - because it was so in the Dockerfile? We can leave it out. When the makefile is invoked, this should be $PWD

Reply:
Done